pkg/types/types.go

```diff
@@ -32,6 +32,7 @@ type ResourceConfig struct {
 		Vendors []string `json:"vendors,omitempty"`
 		Devices []string `json:"devices,omitempty"`
 		Drivers []string `json:"drivers,omitempty"`
+		NetDevs []string `json:"netDevs,omitempty"`
```

Comment:
This is probably a minor thing, but "netDevs" is not intuitive from user perspective in my opinion. How about "ifnames" as in short for "Interface name"? Also it is worth updating the README file under selector section.

Reply:
Or does the ``NetDevs`` intentionally applied to net device only, not includes user space device like uio or vfio?

Reply:
yes

Reply:
Perhaps using ``NetDevNames``If it's for kernel network device only?